test/README.docker.md

```diff
@@ -0,0 +1,17 @@
+Testing in Docker
+=================
+
+## Building a Docker Image for Testing
+
+From the top-level gratia-probe directory, build the image with:
+
+```
+docker build -t gratia-probe-testing -f test/Dockerfile .
+```
+
+And run with:
+
+```
+docker run -it gratia-probe-testing
```

Comment:
@edquist what are you imagining as the dev workflow for this? Will a developer rebuild the container each time they make changes then run the script they're testing directly with something like this?

```
docker run -it gratia-probe-testing /usr/share/gratia/condor/condor_meter
```

What do you do if you need to check logs?

Reply:
I'd like this `README.md` to be a bit more fleshed out (think about how you would explain how to use this to a student) but it can go in a separate PR.